### DISCUSSION TITLE
Not as usable as the website

### DISCUSSION THREAD

**Discussion by j-romchain** [community/community#36426] [Mobile]
Everywhere, there are things I can access via the website that the app cannot open, including but not limited to the releases tab and the actions tab. Due to this, I would like a temporary solution at least to add a option in the top right menu to open the current page in the browser instead of the app while such tabs are currently unaccessible via the app.

**Comment by iskae** [comment]
Hi @j-romchain 👋 

Thanks so much for reaching out ❤️  We have been supporting a share action in a few select features, which can be used for opening the current screen in a browser, but since we are not covering every possible feature with that, I will be creating an internal issue for discussing this further 🙇 

Please stay tuned for our future releases 😊 

Cheers!

**Reply by j-romchain** [reply]
thank you, I mention a specific button because on many android devices (Samsung specifically), you cannot simply select the browser app when you select share. in order to open in browser you have to share to a text with yourself, and then open up the link that you sent to yourself...

**Comment by ossanna16** [comment]
Hey there @j-romchain and thank you for the feedback! I'm glad you found our community 🙂
